thirdparty/raknet/SocketLayer.cpp

```diff
@@ -24,6 +24,10 @@
 #include <netdb.h>
 #endif
 
+#if defined( __HAIKU__ )
```

Comment:
```suggestion
#ifdef __HAIKU__
```

Reply:
there's no use change it because raknet contains both ifdefs and if defineds